### PR TITLE
feat: add new producer version

### PR DESCRIPTION
**Pull request by pritamstyz4ever**
## Context

New producer version has some API changes

## Objective

This PR updates usage for producer service API 

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by jithine — APPROVED** [review]

File: plugins/worker/lib/jobs.js [hunk 1]
```diff
@@ -136,12 +136,13 @@ async function pushToRabbitMq(message, queue, messageId) {
...
+ * @param {String} messageId The message id
  */
-async function pushToKafka(message, topic) {
-    const conn = await AWSProducer.connect();
+async function pushToKafka(message, topic, messageId) {
+    const producer = await AWSProducer.connect();
 
-    if (conn) {
-        await AWSProducer.sendMessage(message, topic);
+    if (producer) {
```
Comment (sagar1312) [inline plugins/worker/lib/jobs.js:144]: `pushToKafka` function is invoked only when kafka is enabled. In that case `AWSProducer.connect()` must return producer object (non null).
Why would we need this check?
Is it possible that `AWSProducer.connect()` would resolve to `null` even when kafka config is enabled. If yes, isn't that an undesired behavior?

Comment (pritamstyz4ever) [reply plugins/worker/lib/jobs.js:144]: it can be null, if some setting are wrong, but queue service should not crash

Comment (sagar1312) [reply plugins/worker/lib/jobs.js:144]: Should we log error in this case to detect and act on this?

Comment (pritamstyz4ever) [reply plugins/worker/lib/jobs.js:144]: it will be logged as part of the package

**Pull request merged**